src/ocean/io/serialize/StructSerializer.d

```diff
@@ -32,6 +32,8 @@ import ocean.core.Exception;
 
 import ocean.io.model.IConduit: IOStream, InputStream, OutputStream;
 
+import ocean.io.serialize.TypeId : DropTypedef;
```

Comment:
https://github.com/sociomantic-tsunami/ocean/blob/v3.x.x/src/ocean/meta/types/Typedef.d#L143

Reply:
I'd like to avoid an extra template instantiation when a cheap is-Expression does the trick.
also further up in this file the same is used.